### ISSUE TITLE
AutoHeal Detector

### ISSUE THREAD

**Issue by stevenernst131** [ShekharGupta1988/AppLensV2#364]
Once ANT60 is done, Auto Heal will finally have events in kusto. 

Though they are not causes of downtime (hopefully), for internal purposes, CSS is very interested in when these happen

**Comment by PraveenTB** [comment]
Consult with Andrey's restart detector. See if this can be folded into it or if there are scenarios that are outside of restart detector
